HW_3/task_1.py

```diff
@@ -0,0 +1,18 @@
+"""Задача 16: Требуется вычислить, сколько раз встречается некоторое
+число X в массиве A[1..N]. Пользователь в первой строке вводит
+натуральное число N – количество элементов в массиве. В последующих
+строках записаны N целых чисел Ai
+. Последняя строка содержит число X"""
+
+import random
+n = int(input("Введите какое количество элементов будет в массиве:"))
+A = 1
+B = 10
+Array = [random.randint(A, B) for _ in range(n)]
+print(Array)
+x = int(input("Введите число, которое будет подсчитано сколько раз встречается в массиве:"))
+count = 0
+for i in range(0, len(Array)):
```

Comment:
Во-первых, в функции `range` ноль передавать первым аргументом не обязательно. Во-вторых, можно напрямую проходиться по элементам списка.
```suggestion
for i in Array:
```